kubectl-openebs/cli/command/get/volume_list.go

```diff
@@ -32,39 +31,28 @@ This command displays status of available zfs Volumes.
 If no volume ID is given, a list of all known volumes will be displayed.
 
 Usage: kubectl openebs cStor volume list [options]
-	`
-	namespace string
+`
 )
 
-const (
-	volumeStatusOK = "Running"
-)
-
-// NewCmdVolumesList displays status of OpenEBS Volume(s)
-func NewCmdVolumesList() *cobra.Command {
+// NewCmdGetVolume displays status of OpenEBS Volume(s)
+func NewCmdGetVolume() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "list",
-		Short: "Displays status information about Volume(s)",
-		Long:  volumesListCommandHelpText,
+		Use:     "volume",
+		Aliases: []string{"vol", "v", "volumes"},
+		Short:   "Displays status information about Volume(s)",
+		Long:    volumesListCommandHelpText,
 		Run: func(cmd *cobra.Command, args []string) {
 			util.CheckErr(RunVolumesList(cmd), util.Fatal)
 		},
 	}
-
-	cmd.Flags().StringVarP(&namespace, "namespace", "n", "openebs",
-		"namespace name, required if volume is not in the `default` namespace")
-
-	flag.CommandLine.Parse([]string{})
-
```

Comment:
This namespace flag, among with other nice flags from kubectl get included via the k8s.io/cli-runtime package 's genericclioptions, thus this is not required, that being said, a nicer way to move such info is still required so that this isn't done at each sub-command.